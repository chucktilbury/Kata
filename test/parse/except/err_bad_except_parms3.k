;-------------------------------------------------------------------------------
; except parms absent
;-------------------------------------------------------------------------------

func name1()() {

    try {
        name2 = "string"
    }
    except() {
        name3 = 123
    }
}