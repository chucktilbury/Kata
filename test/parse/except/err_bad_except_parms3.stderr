Syntax: /home/chuck/Src/Kata/test/parse/except/err_bad_except_parms3.k: 10: 13: expected a SYMBOL but got )
Syntax: /home/chuck/Src/Kata/test/parse/except/err_bad_except_parms3.k: 11: 16: expected function body element but got =
Syntax: /home/chuck/Src/Kata/test/parse/except/err_bad_except_parms3.k: 13: 2: expected function body but got end of file
